test/is_clamped_test.cpp

```diff
@@ -0,0 +1,224 @@
+﻿//  (C) Copyright TODO
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <boost/algorithm/clamp.hpp>
+#include <boost/algorithm/is_clamped.hpp>
```

Comment:
You should include is_clamped.hpp first, to make it more likely to detect if the file is missing necessary includes. This is general advice for any test - include the file being tested first. Example:
https://github.com/boostorg/json/blob/5b45854bb0422d9a37d846c018535cc1004e1449/test/array.cpp#L10

Reply:
done